src/sign/mod.rs

```diff
@@ -1,10 +1,387 @@
 //! DNSSEC signing.
 //!
 //! **This module is experimental and likely to change significantly.**
-#![cfg(feature = "sign")]
-#![cfg_attr(docsrs, doc(cfg(feature = "sign")))]
+//!
+//! Signatures are at the heart of DNSSEC -- they confirm the authenticity of
+//! a DNS record served by a security-aware name server.  Signatures can be
+//! made "online" (in an authoritative name server while it is running) or
+//! "offline" (outside of a name server).  Once generated, signatures can be
+//! serialized as DNS records and stored alongside the authenticated records.
```

Comment:
This documentation is here to provide some context to users as to how this module should be used.  Instead of removing this, perhaps I should add a second paragraph which explains this more thoroughly?

Reply:
I'd rather drop it, but let's leave it for now as it's not wrong, just my 10c as they say.